MatrixSDK/Background/MXBackgroundSyncService.swift

```diff
@@ -579,13 +579,18 @@ public enum MXBackgroundSyncServiceError: Error {
             return
         }
         
-        olmDevice.addInboundGroupSession(sessionId,
-                                         sessionKey: sessionKey,
-                                         roomId: roomId,
-                                         senderKey: senderKey,
-                                         forwardingCurve25519KeyChain: forwardingKeyChain,
-                                         keysClaimed: keysClaimed,
-                                         exportFormat: exportFormat)
+        MXRoomState.load(from: store, withRoomId: roomId, matrixSession: nil) { [weak self] state in
```

Comment:
This feels a bit awkward but I did not find a better way to get the room state whilst in background sync

Reply:
We can avoid this and async call if we have `historyVisibility` on room summary

Reply:
Yea that does make it better, thanks